src/transaction/log_lsa_utils.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "log_lsa_utils.hpp"
+
+#include "log_lsa.hpp"
+#include "packer.hpp"
+
+#include <cstring>
+
+namespace cublog::lsa_utils
+{
+  void
+  pack (const log_lsa &lsa, cubpacking::packer &serializer)
+  {
+    serializer.pack_bigint (lsa);
```

Comment:
Be specific that a cast is involved.
```suggestion
    serializer.pack_bigint (static_cast<std::int64_t> (lsa));
```